core/constants.yaml

```diff
@@ -54,6 +54,9 @@ dev: &dev
   # Delegation
   lightDlgConfirmationTimeout: 500
 
+  # Transaction resubmition
+  pendingTxResubmissionPeriod: 10 # seconds
+  pendingTxAttemptsLimit: 5
```

Comment:
As I said in PM, we decided to get rid of this limit. Reason: if the transaction has managed to escape to outer network, we cannot longer predict what is going to happen in its lifetime - probably, after 5 attempts, some node is going to include it in block, and its "expired" status is going to be invalid. True tx expiration is possible only if we include expiry slot into core structure of transaction, but we're not going to do it for now.